checker/src/main/java/org/checkerframework/checker/index/upperbound/messages.properties

```diff
@@ -1,6 +1,6 @@
 ### Error messages for the Upper Bound Checker
 array.access.unsafe.high=Potentially unsafe array access: the index could be larger than the array's bound\nfound   : %s\nrequired: @IndexFor("%s") or @LTLengthOf("%s") -- an integer less than %s's length
-array.access.unsafe.high.constant=Potentially unsafe array access: the constant index %s could be larger than the array's bound\nfound   : %s\nrequired: @MinLen("%s") -- an array guaranteed to have at least %s elements
+array.access.unsafe.high.constant=Potentially unsafe array access: the constant index %s could be larger than the array's bound\nfound   : %s\nrequired: @MinLen(%s) -- an array guaranteed to have at least %s elements
```

Comment:
This change seems unrelated to this pull request, though I agree we should make it. Can you separate it into a separate PR?